From e43f0c16a76689f23f9905f66095c7a69b908ad2 Mon Sep 17 00:00:00 2001
From: Jerry Wong <jerrywo@xilinx.com>
Date: Fri, 6 Nov 2020 21:25:20 -0800
Subject: [meta-petalinux][master-next][PATCH] power-advantage-tool: Update
 version of PowerAdvantageTool/jupyter-pat

Power Advantage Tool: Power printout moves VCC_RAM from the System Domain
to the PL Domain.

Corrects error in power printout.

Signed-off-by: Jerry Wong <jerry.wong@xilinx.com>
---
 recipes-utils/power-advantage-tool/power-advantage-tool_git.bb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/recipes-utils/power-advantage-tool/power-advantage-tool_git.bb b/recipes-utils/power-advantage-tool/power-advantage-tool_git.bb
index 1c72962..5c4a231 100644
--- a/recipes-utils/power-advantage-tool/power-advantage-tool_git.bb
+++ b/recipes-utils/power-advantage-tool/power-advantage-tool_git.bb
@@ -12,7 +12,7 @@ inherit jupyter-examples python3-dir
 SRC_URI = "git://gitenterprise.xilinx.com/PowerAdvantageTool/jupyter-pat.git;protocol=https;nobranch=1"
 
 PV = "2.2.0+git${SRCPV}"
-SRCREV = "51878b8d93cb91e26af67b04d46635b212ca93b9"
+SRCREV = "91f71431c887ef6f8f1b473ef1f86055d2e3ec53"
 
 S = "${WORKDIR}/git"
 
-- 
2.17.1

